macOS/fastlane/Fastfile

```diff
@@ -1,4 +1,6 @@
 # frozen_string_literal: true
+require_relative "spaceship_patch"
```

Comment:
Sneaking in [this fix for the recent code signing issue](https://github.com/fastlane/fastlane/pull/29588#issuecomment-2989474271)